src/apiCalls.js

```diff
@@ -20,6 +20,7 @@ const fetchBookingData = () => {
 
 const fetchUser = (id) => {
   return fetch(`http://localhost:3001/api/v1/customers/${id}`)
+    // .then(checkForLoginError(id))
     .then(response => response.json())
     .catch(err => console.error('Something went wrong with user'))
```

Comment:
instead of just console.error for the catch, add a function to display to the user. Turn off your 3001 server and try to log in. You'll need an error on the DOM there. apiCalls should look something like: .catch(domUpdates.displayCatchError())